laterpay/application/Controller/Post.php

```diff
@@ -27,15 +27,15 @@ class LaterPay_Controller_Post extends LaterPay_Controller_Abstract
      */
     public function ajax_load_purchased_content() {
         if ( ! isset( $_GET['action'] ) || $_GET['action'] !== 'laterpay_post_load_purchased_content' ) {
-            exit;
+            wp_die();
```

Comment:
wp_die requires a $message that it renders to the user, see http://codex.wordpress.org/Function_Reference/wp_die